torch/_dynamo/config.py

```diff
@@ -157,13 +156,13 @@
 # the `allowed_functions.is_allowed` function will not consider it
 # when creating a list of PyTorch functions that will appear in
 # FX IR.
-allowed_functions_module_string_ignorelist = {
+allowed_functions_module_string_ignorelist = [
```

Comment:
This was to make the config more determinstic. Please advise if this is ok @voznesenskym @ezyang

Reply:
Relocated to new PR